.env.sample

```diff
@@ -1,21 +1,30 @@
 # PORT
-PORT=
+PORT=3002
 
-# App
+# APP
 NODE_ENV=development
 TZ=Europe/London
 LOG_LEVEL=info
 
-# Swagger
+# SWAGGER
 SWAGGER_USER=
 SWAGGER_PASSWORD=
 
 # API
 API_KEY=
 
-# Microsoft Graph
-GRAPH_BASE_URL=
+# APIM MDM
```

Comment:
We shouldn't need MDM here

Reply:
I have been informed number generator call is required for eStore site creation.

Reply:
Oh! Makes sense then.